index.html

```diff
@@ -38,5 +38,118 @@ <h2 class="text-container" id="sdsection">
+        <ul class="technologies">
+          <li class="tech-tag">Ruby on rails</li>
+          <li class="tech-tag">css</li>
+          <li class="tech-tag">JavScript</li>
+          <li class="tech-tag">html</li>
+        </ul>
+        <button class="see-project">See Project</button>
+      </div>
+      <div class="card">
+        <div class="img-wrapper"><img class="img-placeholder" src="" /></div>
+        <h2 class="card-name">Multi-Post Stories<br>Gain+Glory</h2>
+        <ul class="technologies">
+          <li class="tech-tag">Ruby on rails</li>
+          <li class="tech-tag">css</li>
+          <li class="tech-tag">JavScript</li>
+          <li class="tech-tag">html</li>
+        </ul>
+        <button class="see-project">See Project</button>
+      </div> 
+      <div class="card">
+        <div class="img-wrapper"><img class="img-placeholder" src="" /></div>
+        <h2 class="card-name">Multi-Post Stories<br>Gain+Glory</h2>
+        <ul class="technologies">
+          <li class="tech-tag">Ruby on rails</li>
+          <li class="tech-tag">css</li>
+          <li class="tech-tag">JavScript</li>
+          <li class="tech-tag">html</li>
+        </ul>
+        <button class="see-project">See Project</button>
+      </div> 
+    </div>
+    <section class="about-me">
+      <h2 class="about-me-title">About me</h2>
+      <p class="about-me-description"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.</p>
+      <button class="resume-button">Get My Resume</button>
+    </section>  
+    <ul class="assets-list">  
+    <div class="card-2-container">
+    <li class="card-2">
+      <img class="icon-lang" src="icon-lang.png" />
+      <h2 class="card-name-2">Languages</h2>
+      <ul class="technologies-2">
+        <li class="tech-tag-2">JavaScript</li>
+        <li class="tech-tag-2">Ruby</li>
+        <li class="tech-tag-2">Html</li>
+        <li class="tech-tag-2">CSS</li>
+      </ul>
+    </li>
+    <li class="card-2">
+      <img class="icon-lang" src="icon-frameworks.png" />
+      <h2 class="card-name-2">Frameworks</h2>
+      <ul class="technologies-2">
+        <li class="tech-tag-2" id="tag-0">Bootstrap</li>
+        <li class="tech-tag-2" id="tag-1">Ruby on rails</li>
+        <li class="tech-tag-2" id="tag-2">RSpec</li>
+        <li class="tech-tag-2" id="tag-0">Capybara</li>
+        <li class="tech-tag-2" id="tag-1">Selenium</li>
+      </ul>
+    </li>
+    <li class="card-2">
+      <img class="icon-lang" src="icon-skills.png" />
+      <h2 class="card-name-2">Skills</h2>
+      <ul class="technologies-2">
+        <li class="tech-tag-2">Codekit</li>
+        <li class="tech-tag-2">GitHub</li>
+        <li class="tech-tag-2">Codepen</li>
+        <li class="tech-tag-2">Gitlab</li>
+        <li class="tech-tag-2">Terminal</li>
+      </ul>
+    </li>
```

Comment:
- [ ] Great job with the previous reviewer's suggestion on how to make your page look more impressive in [this suggestion](https://github.com/baezluise/Portfolio/pull/2#discussion_r1073844614) on using the right `padding`/`margin` for your spacing. But it appears that some things are still not right on this last card which is affecting the potential aesthetic design of your portfolio. How about giving it a second look? See how the project technologies are collapsing to the edge of your card - thereby affecting user experience?

![image](https://user-images.githubusercontent.com/99715640/213838509-052d2609-82d7-49af-b40d-436fa3895b93.png)

- Kindly make it look very free and nice as it is in the figma design.